ztag/schema.py

```diff
@@ -304,9 +312,9 @@ class CensysString(WhitespaceAnalyzedString):
             "microsoft_csp_signature": Boolean(),
             "microsoft_root_list_signer": Boolean(),
             "microsoft_system_health_loophole": Boolean(),
-            #"unknown":ListOf(OID()) # TODO
-        }, exclude=["bigquery",], category="Extended Key Usage"), # TODO
-        "certificate_policies":ListOf(certificate_policy, category="Certificate Policies"),
+            "unknown": ListOf(OID(), doc="A list of the raw OBJECT IDENTIFIERs of any EKUs not recognized by the application."),
+        }, exclude=["bigquery",], category="Extended Key Usage", validator=Valid()), # TODO
+        "certificate_policies":ListOf(certificate_policy, category="Certificate Policies", validator=Valid()),
```

Comment:
This would be a good opportunity for `validation_policy='ignore'`.

Reply:
done in https://github.com/zmap/ztag/pull/98/commits/0867dd3fff64817d73c42516fc041b9bd22c699b